src/locales/fr/translation.json

```diff
@@ -41,26 +52,32 @@
   "Infectious": "Infectieux",
   "Infectious period": "Période infectieuse",
   "Initial suspected cases": "Premiers cas suspects",
+  "Isolated": "Isolé",
   "Latency": "Latence",
   "Loading...": "Chargement...",
   "Log scale": "Échelle logarithmique",
-  "Mild": "Doux",
+  "Mild": "Léger",
   "Mitigation": "Atténuation",
   "Modal title": "Titre modal",
+  "Moderate mitigation": "Atténuation modérée",
+  "Moderate/North": "Modéréé/Nord",
+  "Moderate/South": "Modéréé/Sud",
+  "Moderate/Tropical": "Modéréé/Tropiques",
+  "No mitigation": "Pas d'atténuation",
   "No such country in our data": "Aucun pays de ce type dans nos données",
   "Not found": "Introuvable",
-  "Number of people served by health care system.": "Nombre de personnes desservies par le système de santé.",
   "Number of cases imported from the outside per day on average": "Nombre de cas importées de l'extérieur par jour en moyenne",
   "Number of cases present at the start of simulation": "Nombre de cas présents au début de la simulation",
   "Number of controllable points on the mitigation curve": "Nombre de points contrôlables sur la courbe d'atténuation",
   "Number of hospital beds available in health care system. Presets are rough estimates indicating total capacity. Number of beds available for COVID-19 treatment is likely much lower.": "Nombre de lits d'hôpital disponibles dans le système de soins de santé. Les préréglages sont des estimations approximatives indiquant la capacité totale. Le nombre de lits disponibles pour le traitement du COVID-19 est probablement beaucoup plus faible.",
   "Number of ICU/ICMUs available in health care system. Presets are rough estimates indicating total capacity. Number of ICU/ICMUs available for COVID-19 treatment is likely much lower.": "Nombre d'unités de soins intensifs disponibles dans le système de soins de santé. Les préréglages sont des estimations approximatives indiquant la capacité totale. Le nombre d'unités de soins intensifs / ICMU disponibles pour le traitement au COVID-19 est probablement beaucoup plus faible.",
+  "Number of people served by health care system.": "Nombre de personnes desservies par le système de santé.",
```

Comment:
*desservies* seems a bit strange to me, as the health care system isn't harming people :p What do you think of *couvertes* instead?

Reply:
I change for `_prises en charge_`, I think it's more suitable.